### ISSUE TITLE
TT Unloading Map Generators

### ISSUE THREAD

**Issue by Zilacon** [MinimallyCorrect/TickThreading#1156]
2014-05-23 17:17:52 [INFO] Attempted to place a tile entity (c chocolate.mods.BD.block.BlockAltarTileEntity @1035969266 in Resource/0 at x, y, z: -1817, 42, 196) at -1817,42,196 (AIR) where there was no entity tile!
2014-05-23 17:17:52 [INFO] Chunk coordinates: -1824,192
2014-05-23 17:17:52 [SEVERE] java.lang.Exception
2014-05-23 17:17:52 [SEVERE]    at net.minecraft.world.chunk.Chunk.func_76604_a(Chunk.java:1113)
2014-05-23 17:17:52 [SEVERE]    at net.minecraft.world.World.func_72837_a(World.java:940)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.builder.docorator.DecoratorRoomBlock.setTileEntity(DecoratorRoomBlock.java:1221)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.builder.docorator.DecoratorRoomBlock.placeTable(DecoratorRoomBlock.java:984)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.builder.docorator.DecoratorRoomBlock.decorateBlacksmith(DecoratorRoomBlock.java:459)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.builder.docorator.DecoratorRoomBlock.placeDecorationBlock(DecoratorRoomBlock.java:117)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.builder.configurable.DungeonRoom.decorate(BuilderDungeon.java:435)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.builder.configurable.BuilderDungeon.generate(BuilderDungeon.java:66)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.builder.configurable.BuilderDungeon.generate(BuilderDungeon.java:55)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.WorldGeneratorNew.generateBigDungeon(WorldGeneratorNew.java:188)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.WorldGeneratorNew.generateSurface(WorldGeneratorNew.java:146)
2014-05-23 17:17:52 [SEVERE]    at chocolate.mods.BD.WorldGeneratorNew.generate(WorldGeneratorNew.java:52)
2014-05-23 17:17:52 [SEVERE]    at cpw.mods.fml.common.registry.GameRegistry.generateWorld(GameRegistry.java:133)
2014-05-23 17:17:52 [SEVERE]    at net.minecraft.world.gen.ChunkProviderServer.populate(ThreadedChunkProvider.java:665)
2014-05-23 17:17:52 [SEVERE]    at net.minecraft.world.gen.ChunkProviderServer.tryPopulateChunks(ThreadedChunkProvider.java:687)
2014-05-23 17:17:52 [SEVERE]    at net.minecraft.world.gen.ChunkProviderServer.getChunkAtInternal(ThreadedChunkProvider.java:644)
2014-05-23 17:17:52 [SEVERE]    at net.minecraft.world.gen.ChunkProviderServer.getChunkAt(ThreadedChunkProvider.java:518)
2014-05-23 17:17:52 [SEVERE]    at nallar.patched.storage.ThreadedChunkProvider$ChunkLoadRunnable.run(ThreadedChunkProvider.java:948)
2014-05-23 17:17:52 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
2014-05-23 17:17:52 [SEVERE]    at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
2014-05-23 17:17:52 [SEVERE]    at nallar.tickthreading.util.FakeServerThread.run(FakeServerThread.java:16)

And...

http://prntscr.com/3lzmn3

Basically when BetterDungeons tried to generate a dungeon it gets blocked by this i think. Cause im flying around looking for dungeons and i got the spawn chance cranked up to 50% and im seeing this error more then im seeing the dungeons.
